.github/workflows/build.yml

```
@@ -0,0 +1,42 @@
+name: Build and Test
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - master
+
+# Prevent multiple builds from running simultaneously on the same branch
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Well the concurency is bound to this workflow anyway so there's no point adding the workflow in the group def I think.

```suggestion
  group: ${{ github.ref }}
```

---

Good catch, updated.